src/dbt/adapters/deltastream/impl.py

```diff
@@ -212,8 +212,7 @@ def get_columns_in_relation(
                 column_info = DeltastreamColumn(
                     column=row[0],  # column name
                     dtype=row[1],  # data type
-                    mode=row[2],  # mode (nullable or not)
-                    fields=None,
+                    mode= "NULLABLE" if row[2] else "REQUIRED",  # mode (nullable or not)
```

Comment:
Mode was passed as a `bool` instead of `str` and the describe returns the "nullable" value?